src/Summoner/License.hs

```diff
@@ -76,3 +79,10 @@ customizeLicense l t nm year
             (beforeN, withN) = T.span (/= '[') afterY
             afterN           = T.tail $ T.dropWhile (/= ']') withN
         in  beforeY <> year <> beforeN <> nm <> afterN
+
+getLicense :: LicenseName -> IO (Maybe License)
+getLicense name = do
```

Comment:
It's better to name this function as `fetchLicense` or `downloadLicense`. `get*` prefix is too generic.